dev-docs/publisher-api-reference.md

```diff
@@ -613,41 +613,45 @@ See the table below for the list of properties in the `mediaTypes` object of the
 For an example of a native ad unit, see below.  For more detailed instructions, see [Show Native Ads]({{site.baseurl}}/dev-docs/show-native-ads.html).
 
 ```javascript
-pbjs.addAdUnits({
-    code: slot.code,
-    sizes: slot.size,
-    mediaTypes: {
-        native: {
-            image: {
-                required: true
-            },
-            title: {
-                required: true,
-                len: 80
-            },
-            sponsoredBy: {
-                required: true
-            },
-            clickUrl: {
-                required: true
-            },
-            body: {
-                required: true
-            },
-            icon: {
-                required: true
+    pbjs.addAdUnits({
+        code: slot.code,
+        sizes: slot.size,
```

Comment:
adUnit.sizes should be ignored on native ad units.

Reply:
Oops - thanks.  Fixed in 081cbee